analytics/spark/src/main/java/mil/nga/giat/geowave/analytic/spark/GeoWaveRDD.java

```diff
@@ -10,7 +10,7 @@
 public class GeoWaveRDD implements
 		Serializable
 {
-	private JavaPairRDD<GeoWaveInputKey, SimpleFeature> rawRDD = null;
+	protected JavaPairRDD<GeoWaveInputKey, SimpleFeature> rawRDD = null;
```

Comment:
this is minor, but I can't say I see the reasoning for changing this to protected